### DISCUSSION TITLE
Dates vs datetimes in the hubverse

### DISCUSSION THREAD

**Discussion by annakrystalli** [hubverse-org/.github#31] [General]
A recent bug report in [#118](https://github.com/hubverse-org/hubValidations/issues/118) where folks submitting datetime columns where they should have been submitting dates was causing an execution error (fixed in https://github.com/hubverse-org/hubValidations/pull/119) prompted me to notice that:
- Apart from the schema for specific task IDs we don't really mention that we currently only support dates, specifically in ISO format and not datetimes across the hubverse.
- There is also nothing stopping admins from adding their own custom taskIDs where they might expect data times (although it's hard to imagine how they would encode datetime valid values in the config). In any case, our framework currently cannot support handling of datetimes. This would require us to add mappings between json datetime data type to R datetime datatype and finally to the arrow datetime datatype equivalent.

So my questions are:

1. Should we add a specific section on dates somewhere in `hubDocs`? Any preferences for where?
2. Are datetimes sth we should support? If not, should we make that clear in the aforementioned docs section?

**Comment by nickreich** [comment]
Some consensus at the hub devteam meeting today to stick with just ISO dates for now. 

@bsweger says "I propose that we support multiple time zones". I think we do this already by accepting the timezone as a field in the admin config.

Someone was going to create a documentation issue about accepting dates only and tag it here and close this discussion.

**Reply by zkamvar** [reply]
Opened in https://github.com/hubverse-org/hubDocs/issues/177
